libs/vscode-ui/components/src/lib/multiple-select/multiple-select.component.ts

```diff
@@ -19,21 +17,23 @@ import { Subscription } from 'rxjs';
   styleUrls: ['./multiple-select.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class MultipleSelectComponent implements OnInit, OnChanges, OnDestroy {
+export class MultipleSelectComponent implements OnInit, OnDestroy {
   @Input() field: Option;
   @Input() value: string[];
   @Output() readonly valueChange = new EventEmitter<string[]>();
-  selectControl = new FormControl([]);
+  selectControl: FormControl;
   items: string[];
   parentFormGroup: FormGroup;
 
-  constructor(private readonly controlContainer: ControlContainer) {
-    this.parentFormGroup = this.controlContainer.control as FormGroup;
-  }
+  constructor(private readonly controlContainer: ControlContainer) {}
 
   private readonly subscriptioins = new Subscription();
 
   ngOnInit(): void {
```

Comment:
Found a little more cleanup on this form component... In the template, it was setting [formControl] and [formControlName] and formControl was being set to that empty initialized `selectControl = new FormControl([])`. That empty initialized FormControl seems to be why the ngOnChanges was doing the setValue below. No bueno. Now it grabs the form control from the parent form group by name and the value is set with the FormBuilder in the parent TaskExecutionFormComponent.